pkg/cmd/cli/cmd/edit.go

```diff
@@ -195,6 +195,13 @@ func RunEdit(fullName string, f *clientcmd.Factory, out io.Writer, cmd *cobra.Co
 			continue
 		}
 
+		err = resource.NewFlattenListVisitor(updates, rmap).Visit(func(info *resource.Info) error {
```

Comment:
@smarterclayton I believe you mean the `RequireNamespace` visitor func (sorry, this piece is new to me). Fixed, please review.